hera_cal/redcal.py

```diff
@@ -269,6 +337,127 @@ def multiply_by_gains(target, gains, target_type='vis'):
 
     return _apply_gains(target, gains, (lambda x, y: x * y), target_type)
 
+class OmnicalSolver(linsolve.LinProductSolver):
+    def __init__(self, data, sol0, wgts={}, gain=.3, **kwargs):
+        """Set up a nonlinear system of equations of the form g_i * g_j.conj() * V_mdl = V_ij
+        to linearize via the Omnical algorithm from Liu et al. 2010.
```

Comment:
It's not in that paper. Perhaps cite the memo?

Reply:
Done.